mne_bids/tests/test_read.py

```diff
@@ -967,19 +970,58 @@ def test_write_read_fif_split_file(tmp_path):
     raw = mne.io.RawArray(data, raw.info)
     big_fif_fname = pathlib.Path(tmp_dir) / 'test_raw.fif'
     raw.save(big_fif_fname, split_size='10MB')
-    raw = _read_raw_fif(big_fif_fname, verbose=False)
-    write_raw_bids(raw, bids_path, verbose=False)
+
+    # write_raw_bids() doesn't support a `split_size` parameter, and would
+    # normally only start splitting FIFF files exceeding 2GB in size. To
+    # write split files smaller than this, we hack our way around things and
+    # manually write two files with a `split` entity.
```

Comment:
But this means reading the data back by passing the first filename won't automatically read the remaining splits in this test. So this is **very** synthetic.